erase-install.sh

```diff
@@ -90,7 +90,7 @@ dialog_dl_title_nl="macOS downloaden"
 dialog_dl_title_fr="Téléchargement de macOS"
 
 dialog_dl_desc_en="We need to download the macOS installer to your computer; this will take several minutes."
-dialog_dl_desc_de="Der macOS Installer wird heruntergeladen, dies dauert mehrere Minuten."
+dialog_dl_desc_de="Das macOS-Installationsprogramm wird heruntergeladen; dies kann einige Minuten dauern."
```

Comment:
Ist "einige" richtig? Es kann 30+ Minuten dauern? Wohl bin ich keinen Deutsch-Experte, aber "einige" in meine Meinung heisst < 10 Minuten.

Reply:
So i really had to check the "Duden" (from the macOS Dictionary.app), because i wasn't completely sure about that. But both words are "unbestimmte Zahlwörter" and they do not specify any hard numeric range; it all depends very much on the context in which they are used: "einige Male angerufen" would be having called 2-5 times, "einigen Ärger bekommen" would be getting in really deep trouble, when Deutsche Bahn says "der Zug hat einige Minuten Verspätung" this could even mean two hours – while with SBB this would probably be at most 3 minutes?

I feel that "einige" fits a little bit better here than "mehrere", because it is a little more vague in this case. With "mehrere" i somehow see distinct minutes before me which one could count individually, while "einige" says "ah, you know, just like it's always with those updates, get a coffee and come back, then it's probably ready".

Reply:
OK, I'll let it in :) Hopefully nobody comes back with "Einige!? Es war Viele!" :-)